src/main/java/com/hiqes/andele/Andele.java

```diff
@@ -246,8 +246,8 @@ public static void checkAndExecute(Fragment fragment, ProtectedAction action) {
      *                         the ActionCallback to execute when granted, etc.
      */
     @SuppressWarnings({"WeakerAccess", "unused"})
-    public static void checkAndExecute(android.support.v4.app.Fragment fragment, ProtectedAction action) {
```

Comment:
My concern with doing this is now the library will not work with codebases which still utilize `android.support.v4`.  It would be better to define new APIs for the `androidx` variants while still leaving the older support lib based APIs in place.  That would effectively allow the library to support both older codebases and new ones.

Reply:
Both are supported now.